nuts-network/private-transactions/prepare.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
+
+set -e
+
+source ../../util.sh
+
+function setupNode() {
+  local did=$(printf '{
+    "selfControl": true,
```

Comment:
don't they need a keyAgreement key for the encryption to work? Or is that one true by default? Then there's a mix of param settings and default settings?

Reply:
It's true by default. These are the default options:


```go
AssertionMethod:      true,
Authentication:       false,
CapabilityDelegation: false,
CapabilityInvocation: true,
KeyAgreement:         true,
SelfControl:          true,
```

Reply:
Added it to the request to make it more clear